src/server.ts

```diff
@@ -17,7 +21,19 @@ app.use(express.json()); //add JSON body parser to each following route handler
 app.use(cors()); //add CORS support to each following route handler
 
 app.get("/", async (_req, res) => {
```

Comment:
The API route should be called /pastes or /snippets. So that you can adhere to RESTful API naming convention